pages/heft_tasks/webpack.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: page
+title: '"webpack" task'
+navigation_source: docs_nav
+---
+
+🚨 *The webpack task is very new. Some features are not yet implemented.* 🚨
+
+<!-- No we are not going to use branded capitalization like "webpack" or "npm". ;-) -->
+
+This task invokes the [Webpack](https://webpack.js.org/) tool to produce application bundles.  It provides features such as:
+
+- Combining many small .js files into one large file for faster downloads
+- Improving performance by applying various compiler optimizations such as inlining and dead code elimination ("tree shaking")
+- Compacting and obfuscating code by shortening identifiers ("uglifying")
```

Comment:
Webpack's default minifier is now [Terser](https://terser.org/), so "uglify" might not be a relevant term anymore.